mscore/menus.cpp

```diff
@@ -1466,9 +1466,15 @@ PalettePanel* MuseScore::newLinesPalettePanel()
 
       TextLine* textLine = new TextLine(gscore);
       textLine->setLen(w);
-      textLine->setBeginText("VII");
```

Comment:
Just curious
Why was it here? :)

Reply:
For a Capo/Barre ? I think it should stay.

Reply:
Using a Roman Chord identifier in a trill line doesn't make much sense, does it?

Reply:
It is not a trill line, or is it? It is (supposed to be) a Capo line, the Roman number supposed to denote the Fret you should place the capo at, as far as I can tell

Reply:
It ia __after__ the trill lines...

Reply:
Sorry, I misread the location.
In the old textline (which was on the staff only), there was this text <code>VII</code>. But now we have the same element which can be both be system and non-system but they have their own entry on the palette. So the text <code>VII</code> is changed to <code>Staff</code> to indicate a staff textline. The new system textline will have the text <code>System</code>. The old <code>VII</code> had no real meaning.